db/upgrade.php

```diff
@@ -148,5 +148,17 @@ function xmldb_cms_upgrade($oldversion) {
         upgrade_mod_savepoint(true, 2023080400, 'cms');
     }
 
+    if ($oldversion < 2023080900) {
+        $table = new xmldb_table('cms_types');
+        $field = new xmldb_field('title_mustache', XMLDB_TYPE_TEXT, null, null, false, null, null, 'datasources');
+
+        // Conditionally launch add field.
+        if (!$dbman->field_exists($table, $field)) {
+            $dbman->add_field($table, $field);
+        }
```

Comment:
should also add migrate task here to set all of the templates to something. Doens't really matter that, just the name of the template would be a good default

Reply:
done